dbcon/dmlpackageproc/commandpackageprocessor.cpp

```diff
@@ -430,7 +430,11 @@ CommandPackageProcessor::processPackage(dmlpackage::CalpontDMLPackage& cpackage)
 
                 if (!cpInvalidated)
                 {
-                    fDbrm->invalidateUncommittedExtentLBIDs(0, &lbidList);
+		    // XXX: HACK!!!HACK!!!HACK!!!
```

Comment:
I like the comment but what it is about? Why do we need a hack and what would be a less hacky solution in your opinion?

Reply:
I explained in the comment.

Basically, it is not clear to me is it good to dispatch on the command string.